pkg/kubecost/summaryallocation.go

```diff
@@ -1237,19 +1243,18 @@ func (sas *SummaryAllocationSet) TotalEfficiency() float64 {
 	sas.RLock()
 	defer sas.RUnlock()
 
-	totalRAMCostEff := 0.0
-	totalCPUCostEff := 0.0
 	totalRAMCost := 0.0
 	totalCPUCost := 0.0
 	for _, sa := range sas.SummaryAllocations {
-		totalRAMCostEff += sa.RAMEfficiency() * sa.RAMCost
-		totalCPUCostEff += sa.CPUEfficiency() * sa.CPUCost
+		if sa.IsIdle() {
+			continue
+		}
```

Comment:
I just want to double-check... do we want to remove this or not? Seems like the real bug fix was the final computation on line 1257. Are we sure that we want to include the idle RAM and CPU cost in the total costs? I thought we were trying to not let idle affect the numbers?